monitoring/grafana/KubeStellarControllersMonitoring.json

```diff
@@ -181,10 +156,10 @@
             "uid": "prometheus-test"
           },
           "editorMode": "code",
-          "expr": "histogram_quantile(0.75, sum by(instance, name, le) (rate(workqueue_queue_duration_seconds_bucket{namespace=\"$SpaceNS\", job=\"$KSController\"}[5m])))",
+          "expr": "histogram_quantile(0.25, sum by(instance, name, le) (rate(workqueue_queue_duration_seconds_bucket{namespace=\"$SpaceNS\", job=\"$KSController\"}[5m])))",
```

Comment:
That 5 minute quantity should be noted in the panel's description.

Reply:
Fixed